<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SBMLLoadDlg</class>
 <widget class="QDialog" name="SBMLLoadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure SBML file data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Path to SBML Model</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameLE"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browsePB">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Model Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="modelNameLE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Model Nickame</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="modelNicknameLE">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This is usually abreviated version of Model Name but can be the same as Model Name. E.g. &lt;span style=&quot; font-weight:600;&quot;&gt;DeltaNotch&lt;/span&gt; model can be abreviated &lt;span style=&quot; font-weight:600;&quot;&gt;DN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leaveEmptyPB">
       <property name="text">
        <string>Leave Empty</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>SBMLLoadDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leaveEmptyPB</sender>
   <signal>clicked()</signal>
   <receiver>SBMLLoadDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
